src/Data/Managers/V2/Environment/ClearEnvironment/useClearEnvironment.ts

```diff
@@ -1,29 +1,20 @@
-import {
-  UseMutationOptions,
-  UseMutationResult,
-  useMutation,
-  useQueryClient,
-} from "@tanstack/react-query";
-import { useDelete } from "../../helpers";
+import { UseMutationOptions, UseMutationResult, useMutation } from "@tanstack/react-query";
+import { useDeleteWithoutEnv } from "../../helpers";
 
 /**
  * React Query hook for clearing (decommissioning) an environment.
  *
  * @returns {UseMutationResult<void, Error, string, unknown>} The mutation object for clearing an environment.
  */
 export const useClearEnvironment = (
-  options?: UseMutationOptions<void, Error, string>
-): UseMutationResult<void, Error, string> => {
-  const client = useQueryClient();
-  const deleteFn = useDelete();
+  environmentId: string,
+  options?: UseMutationOptions<void, Error, void>
```

Comment:
just to understand this better, why did we move from string to void?

Reply:
third parameter is the params of the Mutation, in case of lack of it used void